lib/tests/streamlit/hashing_test.py

```diff
@@ -357,6 +358,14 @@ def test_file_position(self):
             f.seek(0)
             self.assertEqual(h1, get_hash(f))
 
+    def test_pycapsule(self):
+        # get pycapsule instance from torchvision internals
+        capsule = torchvision.models.__dict__["__builtins__"]
+        capsule = dict(capsule)["__pybind11_internals_v3__"]
+
+        assert is_type(capsule, "builtins.PyCapsule")
```

Comment:
Why is this assert needed?

Reply:
Because we're grabbing `capsule` from the internals of the Torchvision model instead of creating it ourselves. The Torchvision implementation could change in the future.